build.sh

```diff
@@ -21,7 +21,8 @@ fi
 
 if [ -z "$1" ]
 then
-  PKG_LIST=`ls */PSPBUILD | xargs dirname`    # By default build them all
+  # By default build them all
+  PKG_LIST=$(find . -name "PSPBUILD" -exec sh -c 'echo $(basename $(dirname $0))' {} \;)
```

Comment:
I think you could also use `find . -name PSPBUILD -execdir pwd \;` here. Should be a bit more readable.

Reply:
I believe it will only work with GNU find and then should still be piped into `dirname`. My version will work with BusyBox (e.g. Alpine in pspdev docker image) and macOS, though it's a bit longer and not as readable.

Reply:
I see, you're right.